### ISSUE TITLE
Available disk space?

### ISSUE THREAD

**Issue by JhaJug** [badabing2005/PixelFlasher#209]
I installed Pixelflasher on a newly installed Kubuntu 24.04 LTS laptop. Every time I try to flash my phone (pixel 6), i get the message : Please check available disk space, you do not have safe levels of available storage to flash without risk. 
It is not possible that my disk space is too small. What is wrong? (Email jhajug7@gmail.com)
[support_2024-06-15_12-07-20.zip](https://github.com/user-attachments/files/15846518/support_2024-06-15_12-07-20.zip)

**Comment by badabing2005** [comment]
```
Available Free Memory: 5.79 GB / 7.54 GB
Available Free Disk on system drive: 3 GB
Available Free Disk on PixelFlasher data drive: 208 GB
```

Your system drive is low in space, make sure it is above 5GB

But I don't understand why you're flashing you're already at `AP2A.240605.024` and trying to flash `oriole-ap2a.240605.024-factory-11cc7d9e.zip`

**Comment by JhaJug** [comment]
Tanks for your quick reply. I did not know that PF takes a such lot of disk space. I will see what I can do to my system drive. As to the double flashing: I already have flashed my phone with the newest factory image on a windows system. I only wanted to make a dry run to see how it works on linux.
BlueMail <https://bluemail.me> for Desktop herunterladen
badabing2005 wrote:
Available Free Memory: 5.79 GB / 7.54 GB Available Free Disk on system drive: 3 GB Available Free Disk on PixelFlasher data drive: 208 GB 
Your system drive is low in space, make sure it is above 5GB
But I don't understand why you're flashing you're already at AP2A.240605.024 and trying to flash oriole-ap2a.240605.024-factory-11cc7d9e.zip

**Comment by badabing2005** [comment]
The Pixel Images are over 3GB, and they need to be unpacked, so you should understand why the space is needed.
